libraries/stdlib/samples/test/samples/collections/reversedViews.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2010-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package samples.collections
+
+import samples.Sample
+import samples.assertPrints
+
+class ReversedViews {
+    @Sample
+    fun asReversedList() {
+        val original = ('a'..'e').toList()
+        val reversed = original.asReversed()
+
+        assertPrints(original, "[a, b, c, d, e]")
+        assertPrints(reversed, "[e, d, c, b, a]")
```

Comment:
It should be shown, how changes in `original` affect `reversed` list in case if `original` is a mutable list.
Like:
```
val original = mutableListOf(...)
val originalReadOnly = original as List<...>
val reversed = originalReadOnly.asReversed()
// reversed is read-only list, but can be changed by modifying original one
```